package.json

```diff
@@ -169,12 +164,7 @@
     "branch-comparer": "^1.1.0",
     "concurrently": "^8.2.2",
     "cross-env": "^7.0.3",
-    "eslint": "^8.57.0",
-    "eslint-config-standard": "^17.1.0",
-    "eslint-import-resolver-node": "^0.3.9",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-n": "^16.6.2",
-    "eslint-plugin-promise": "^6.1.1",
+    "eslint": "^9.0.0",
```

Comment:
Is including eslint separately necessary?

Reply:
Yes, it's how flat config works. However it has the added benefits that this system would work with any eslint-compatible editor out of the box